### PR TITLE
doc: fixing broken md links

### PR DESCRIPTION
**Pull request by joesepi**

### CODE REVIEW

**Review by tobie — CHANGES_REQUESTED** [review]
I think this fixes some but creates some new issues.

File: conduct/COC_TEAM_CHARTER.md [hunk 1]
```diff
@@ -6,38 +6,38 @@ If you want to report an incident, go to [code-of-conduct.openjsf.org][CoC].
 
 # Code of Conduct Team Charter
 
-The Cross Project Council (CPC) delegates enforcement of the [Code of Conduct][CoC] to the Code of Conduct Team (CoC Team) through this Code of Conduct Team Charter. This charter defines the composition and roles of the CoC Team, along with CoC team responsibilities and operations.
+The Cross Project Council (CPC) delegates enforcement of the [Code of Conduct](CoC) to the Code of Conduct Team (CoC Team) through this Code of Conduct Team Charter. This charter defines the composition and roles of the CoC Team, along with CoC team responsibilities and operations.
```
Comment (ljharb) [inline conduct/COC_TEAM_CHARTER.md:9]: this was already correct; it's pointing to the footnote

File: conduct/COC_TEAM_CHARTER.md [hunk 2]
```diff
@@ -6,38 +6,38 @@ If you want to report an incident, go to [code-of-conduct.openjsf.org][CoC].
...
 ## Responsibilities
 
 The CoC Team's responsibilities are to:
 
-- Handle code of conduct [incident reports][incident report].
-- Handle code of conduct [escalation requests][escalation].
-- Handle code of conduct [appeals][appeal].
+- Handle code of conduct [incident reports](incident report).
+- Handle code of conduct [escalation requests](escalation).
+- Handle code of conduct [appeals](appeal).
```
Comment (ljharb) [inline conduct/COC_TEAM_CHARTER.md:17]: as were all these

**Review by tobie — APPROVED** [review]
Fixed

**Review by ljharb — APPROVED** [review]

**Pull request merged**